### PR TITLE
Add Selectour travel agency

### PR DESCRIPTION
**Pull request by bmillemathias**
SSIA

### CODE REVIEW

**Comment by bhousel** [comment]
Thanks @bmillemathias!
Can you also add a tag for

File: brands/shop/travel_agency.json [hunk 1]
```diff
@@ -83,6 +83,15 @@
       "shop": "travel_agency"
     }
   },
+  "shop/travel_agency|Selectour": {
+    "countryCodes": ["fr"],
+    "tags": {
+      "brand": "Selectour",
+      "brand:wikidata": "Q3478073",
+      "name": "Selectour",
```
Comment (bhousel) [inline brands/shop/travel_agency.json]: also add `"brand:wikipedia": "fr:Selectour"`

Comment (bmillemathias) [reply brands/shop/travel_agency.json]: done, it was supposed to be in the patch.
Thanks for spotting that.

**Comment by bhousel** [comment]
Thanks!

**Pull request merged**